linera-web/src/lib.rs

```diff
@@ -54,9 +52,12 @@ async fn get_storage() -> Result<WebStorage, <linera_views::memory::MemoryStore
     .await
 }
 
-type PersistentWallet = persistent::Memory<Wallet>;
-type PersistentSigner = persistent::Memory<InMemorySigner>;
-type ClientContext = linera_client::client_context::ClientContext<WebEnvironment, PersistentWallet>;
+/// A wallet that stores the user's chains and keys in memory.
+#[wasm_bindgen(js_name = "Wallet")]
+pub struct InMemoryWallet(persistent::Memory<Wallet>);
```

Comment:
The name of this struct is purposely more general than its (opaque) implementation because it shouldn't remain an in-memory wallet.

Reply:
It is in memory though.

Reply:
Currently it is, but I would like it to not be. It will be stored in IndexedDB soon.

Reply:
Done.